packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,56 @@
+
+%global repo_orgname RedHatInsights
+%global repo_name insights-client-role
+%global repo_version 1.5
+
+%global role_orgname %{repo_orgname}
+%global role_name insights-client
+%global role_version %{repo_version}
+
+Name: ansiblerole-insights-client
+Summary: Packaging of the insights-client Ansible role
+Version: %{role_version}
+Release: 1%{?dist}
+
+License: ASL 2.0
+
+Source: https://github.com/%{repo_orgname}/%{repo_name}/archive/v%{repo_version}.tar.gz#/%{role_name}-%{role_version}.tar.gz
+
+Url: https://github.com/%{repo_orgname}/%{repo_name}/
+BuildArch: noarch
+
+Requires: ansible
+
+%description
+This package installs the insights-client Ansibile role.
+
+Make sure that "/usr/share/ansible/roles" is on your Ansible role_path.
+
+%prep
+%setup -qc
```

Comment:
Could you add a newline after this to separate sections?

Reply:
Yes.  Will do.

Reply:
fixed